video/image_writer.c

```diff
@@ -163,9 +164,14 @@ static void prepare_avframe(AVFrame *pic, AVCodecContext *avctx,
     );
 }
 
-static bool write_lavc(struct image_writer_ctx *ctx, mp_image_t *image, const char *filename)
+static bool write_lavc(struct image_writer_ctx *ctx, mp_image_t *image,
+                       const char *filename, bool overwrite)
 {
-    FILE *fp = fopen(filename, "wb");
+    FILE *fp = fopen(filename, overwrite ? "wb" : "wbx");
+    if (!fp && errno == EEXIST) {
+        MP_ERR(ctx, "File '%s' already exists!\n", filename);
```

Comment:
Is there something against calling `strerror`? You can just do

```c
        MP_ERR(ctx, "Failed to open file '%s': %s\n", filename, strerror(errno));
```

And it will print "file exists" or something similar for `EEXIST` while giving more info on failure cause in other failure cases as well.